### DISCUSSION TITLE
Fullscreen Android app

### DISCUSSION THREAD

**Discussion by mikegoatly** [AvaloniaUI/Avalonia#15614] [Q&A]
Hi folks, I'm looking at porting an application to Avalonia. On Android I'd like the notification bar to get out of the way, and I've been playing around trying to switch the configuration up in the `OnCreate` method:

```csharp
[Activity(
    Label = "MyApp",
    Theme = "@style/MyTheme.NoActionBar",
    Icon = "@drawable/icon",
    MainLauncher = true,
    ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize | ConfigChanges.UiMode)]
public class MainActivity : AvaloniaMainActivity<App>
{
    protected override void OnCreate(Bundle savedInstanceState)
    {
        base.OnCreate(savedInstanceState);

        // This was the first attempt
        Window.DecorView.SystemUiFlags = (SystemUiFlags)(
            SystemUiFlags.LayoutStable
            | SystemUiFlags.LayoutFullscreen
            | SystemUiFlags.Fullscreen);

        // This also didn't work
        Window.AddFlags(WindowManagerFlags.Fullscreen);
    }

    protected override AppBuilder CustomizeAppBuilder(AppBuilder builder)
    {
        return base.CustomizeAppBuilder(builder)
            .WithInterFont();
    }
}
```

But nothing has an effect. Interestingly any breakpoints I put in there don't get hit, so it's possible that either the debugger isn't attached that early on, or something else is going on.

Has anyone else done something similar?

**Comment by mikegoatly** [comment]
I'm such a fool... I'd disabled rebuilding of the Android project to speed up dev while working on the Windows version of it, that's why the `OnCreate` method wasn't being attached to; it wasn't even being compiled.

**Comment by maxkatz6** [comment]
In general, you can use IInsetManager APIs to do the same, but crossplatform.
https://docs.avaloniaui.net/docs/concepts/services/insets-manager

**Reply by mikegoatly** [reply]
Thanks! I'll have a play with that.
